.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Report a bug or unexpected behavior
+---
+
+<!--
+
+* Filling out this template with the relevant information will help us
+  diagnose and fix the issue. We appreciate the feedback!
+
+* For concerns about data, check the frequently asked questions
+  (http://gnomad-beta.broadinstitute.org/faq) before filing an issue.
+  
+-->
+
+### What you did:
+<!-- What you were doing. Include a link to the page you were viewing. -->
+
+### What happened:
+<!-- Please provide error messages, screenshots, stack traces, etc. -->
```

Comment:
"What happened" and "problem description" sound pretty similar. What if we just put a list of specific things to include
- link to the page
- screensots of the broken parts of the page (+ put the keyboard shortcut for taking a screenshot on a Mac + keyboard shortcut to paste it in - for people who don't know you can paste them into the Issues textbox)
- error messages (+ instructions for how to take a screenshot of the error console in chrome)